services/src/main/java/org/jboss/windup/web/services/rest/RegisteredApplicationEndpointImpl.java

```diff
@@ -20,9 +20,12 @@
     private RegisteredApplicationService registeredApplicationService;
 
     @Override
-    public Collection<RegisteredApplication> getAllApplications()
+    public Collection<RegisteredApplication> getAllApplications(Long projectId)
     {
-        return this.registeredApplicationService.getAllApplications();
+        if (projectId == null)
```

Comment:
Yeah, I think putting it in here is ok, but it probably should be something like getApplicationsByProject, instead of overriding "getAllApplications".

The other paths are used, but they could be changed. :)